test/snippet/core/type_traits/lazy.cpp

```diff
@@ -0,0 +1,34 @@
+#include <vector>                               // std::vector
+#include <forward_list>                         // std::forward_list
```

Comment:
Order lexicographically and put a new line between stl and seqan3 includes.

Reply:
We put the space there also in snippets?

Reply:
I skimmed through some snippets and we do if we have stl includes. But I didn't check all.

Reply:
(I changed it)